UPGRADING.md

```diff
@@ -135,7 +135,7 @@ window.Turbolinks = {
 
         // Old mobile adapters use visit.location.absoluteURL, which is not available
         // because Turbo dropped the Location class in favor of the DOM URL API
-        const adapterVisitStarted = adapter.visitStarted
+        const adapterVisitStarted = adapter.visitStarted.bind(adapter)
```

Comment:
We're having this issue while trying to migrate to Turbo from our apps in `turbolinks-ios`,when the `visitStarted` method in the adapter is called, `this` is a reference to `window` instead of the adapter itself:
https://github.com/turbolinks/turbolinks-ios/blob/1c891b9ffbb89666a05b7c5b02a8fdd700e92323/Turbolinks/WebView.js#L60-L63